test/system/commissions_test.rb

```diff
@@ -0,0 +1,41 @@
+require "application_system_test_case"
+
+class CommissionsTest < ApplicationSystemTestCase
+  setup do
```

Comment:
Never seen a file like this before :/ Can you tell me about how this was created?